packages/firebase_dynamic_links/firebase_dynamic_links_platform_interface/lib/src/google_analytics_parameters.dart

```diff
@@ -0,0 +1,38 @@
+/// The Dynamic Link analytics parameters.
+class GoogleAnalyticsParameters {
+  const GoogleAnalyticsParameters({
+    required String this.campaign,
```

Comment:
Just double checking; these are required fields but still nullable?

Reply:
Not required. Double checked docs and our native setup. All optional. Will remove.